packages/cogify/src/cogify/cli/cli.cog.ts

```diff
@@ -106,11 +107,11 @@ export const BasemapsCogifyCreateCommand = command({
 
       // The cog was already created but deemed invalid
       if (invalidReason) {
-        logger.warn({ item: f.item.id, asset: cogAsset.href, reason: invalidReason }, 'Cog:Create:Exists:invalid');
```

Comment:
this could be solved with `cogAsset?.href` ?

Reply:
That works as well, just fixed.